website/docs/d/scc_posture_profiles.html.markdown

```diff
@@ -0,0 +1,81 @@
+---
+layout: "ibm"
+page_title: "IBM : ibm_scc_posture_profiles"
+description: |-
+  Get information about list_profiles
+subcategory: "Security and Compliance Center"
+---
+
+# ibm_scc_posture_profiles
+
+Provides a read-only data source for list_profiles. You can then reference the fields of the data source in other resources within the same configuration using interpolation syntax.
```

Comment:
The description should have full name, can be rewritten as

Review information about Security and Compliance Center list profiles. For more information, about list profiles, see []().